Makefile

```diff
@@ -7,7 +7,7 @@ deps:
 	docker-compose run --rm cli composer install
 
 depsupdate:
-	docker-compose run --rm cli bash -c "composer update && composer show -Df json > versions.json"
+	docker-compose run --rm cli bash -c "git config --global --add safe.directory /data/vendor/silinternational/yii2-codeception && composer update && composer show -D > versions.txt"
```

Comment:
@Baggerone I don't see an error message when I run `make depsupdate` without the "safe.directory" command. Could you share what the error message was?

Reply:
I don't get the error anymore. So, I've backed out that config add part.